### PR TITLE
Add T2.5 in Amphibious Lab.

### PR DESCRIPTION
**Pull request by skymyj**

### CODE REVIEW

File: units/asubpen.lua [hunk 1]
```diff
@@ -58,12 +58,12 @@ return {
...
-			[4] = "armcroc",
-			[5] = "armjeth",
-			[6] = "armaak",
-			[7] = "armsub",
+ 			[2] = "armmarv",
+			[3] = "armjeth",
+			[4] = "armaak",
+			[5] = "armpincer",
+			[6] = "armcroc",
+			[7] = "armscpion.lua",
```
Comment (nixtux) [inline units/asubpen.lua]: .lua fix it skymyj

**Comment by nixtux** [comment]
What the reason for sub removal?

**Comment by skymyj** [comment]
Cause a Sub is not an amphibious unit. 
If you want I add like before, I will.

**Comment by skymyj** [comment]
Adding Crawling Bomb & Cor/Arm Sub

**Pull request merged**